03-infrastructure-tour/README.md

```diff
@@ -22,7 +22,7 @@ debugging interfaces like Flower, Kibana and the RabbitMQ management interface.
 
 In order to be able to go through all of the infrastructure components that an
 Invenio instance is built from, we have to use the `docker-compose.full.yml`
-setup. To bring it up we have to execute the following commands:
+setup. It is used for demonstration purposes, since it allows us to run all components in containers. To bring it up we have to execute the following commands:
```

Comment:
Line breaks (also further down i think)

Reply:
Done, thanks! Some of them are tables inside a code block, those I left unchanged.